PyPDF2/_utils.py

```diff
@@ -162,13 +174,8 @@ def readUntilRegex(stream, regex, ignore_eof=False):
     return name
 
 
-class ConvertFunctionsToVirtualList(object):
+class _VirtualList(object):
```

Comment:
Using the name you used in `2.0.0-dev` branch for this to hopefully make the merge commit a bit easier.